website/src/blogs/how-to-deploy-nuodb-on-kubernetes-with-openebs.md

```diff
@@ -0,0 +1,192 @@
+
+![Connect to MayaOnline](/public/images/blog/connect-to-mayaonline.png)
+
+### Configure cStor Pool
+
+After OpenEBS is installed, cStor pool must be configured. If cStor Pool is not already configured in your OpenEBS cluster, this can be done following the instructions [here.](https://docs.openebs.io/docs/next/configurepools.html) A sample YAML named openebs-config.yaml used for configuring cStor Pool is provided in the Configuration details below. During cStor Pool creation, make sure that the maxPools parameter is set to >=3.
+
+    #Use the following YAMLs to create a cStor Storage Pool.
+    # and associated storage class.
+    apiVersion: openebs.io/v1alpha1
+    kind: StoragePoolClaim
+    metadata:
+      name: cstor-disk
+    spec:
+      name: cstor-disk
+      type: disk
+      poolSpec:
+        poolType: striped
+      # NOTE - Appropriate disks need to be fetched using `kubectl get disks`
+      #
+      # `Disk` is a custom resource supported by OpenEBS with `node-disk-manager`
+      # as the disk operator
+    # Replace the following with actual disk CRs from your cluster `kubectl get disks`
+    # Uncomment the below lines after updating the actual disk names.
+      disks:
+        diskList:
+    # Replace the following with actual disk CRs from your cluster from `kubectl get disks`
+    #   - disk-184d99015253054c48c4aa3f17d137b1
+    #   - disk-2f6bced7ba9b2be230ca5138fd0b07f1
+    #   - disk-806d3e77dd2e38f188fdaf9c46020bdc
+    #   - disk-8b6fb58d0c4e0ff3ed74a5183556424d
+    #   - disk-bad1863742ce905e67978d082a721d61
+    #   - disk-d172a48ad8b0fb536b9984609b7ee653
+    ---
+
+### Create the Storage Class
+
+You must configure a StorageClass to provision a cStor volume on a given cStor pool. StorageClass is the interface through which most of the OpenEBS storage policies are defined. I will be using a StorageClass to consume the cStor Pool that is created using external disks attached on the Nodes. Since NuoDB is a statefulset application, it requires only a single storage replica. Therefore, the cStor volume replicaCount is =1. A sample YAML named openebs-nuodb.yaml used to consume cStor pool with cStoveVolume Replica count as 1 is provided in the configuration details below.
+
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: openebs-nuodb
+      annotations:
+        openebs.io/cas-type: cstor
+        cas.openebs.io/config: |
+          - name: StoragePoolClaim
+            value: "cstor-disk"
+          - name: ReplicaCount
+            value: "1"       
+    provisioner: openebs.io/provisioner-iscsi
+    reclaimPolicy: Delete
+    ---
+
+### Import the NuoDB CE Template
+
+Import the NuoDB CE template *ce-template-persistent.yaml* into OpenShift by navigating to the “Overview” Tab, clicking “Import YAML/JSON,” and running the import.
+![Import the NuoDB CE Template](/public/images/blog/import-the-nuodb-ce-template.png) Import YAML Template
```

Comment:
> Import YAML Template

This is figcaption.
please use this format to include it

```
(***Import YAML Template***)
```

Reply:
cc @Pallavi-PH

Reply:
Yes, we use (***Import YAML Template***) format to display figure caption. Also, we need to display figure caption on the next line without any line space between the image and figure caption. To achieve this, we can bring the text in the next line in the markdown file and give 2 tab spaces next to image link